ubuntu-advantage

```diff
@@ -30,8 +30,20 @@ install_livepatch_prereqs() {
 enable_livepatch() {
     install_livepatch_prereqs
     if ! is_livepatch_enabled; then
-        echo 'Enabling Livepatch with the given token, stand by...'
-        canonical-livepatch enable "$1"
+        if check_snapd_kernel_support; then
+            echo 'Enabling Livepatch with the given token, stand by...'
+            canonical-livepatch enable "$1"
+        else
+            echo "Your currently running kernel ($(uname -r)) is too old to"
```

Comment:
I'd add a global variable like `KERNEL_VERSION=$(uname -r)` and use it here and below